tests/CommunityToolkit.Aspire.Hosting.Azure.DataApiBuilder.Tests/ContainerResourceCreationTests.cs

```diff
@@ -49,4 +40,66 @@ public void AddDataAPIBuilderContainerDetailsSetOnResource()
         Assert.True(resource.TryGetLastAnnotation(out EndpointAnnotation? httpEndpointAnnotations));
         Assert.Equal(5000, httpEndpointAnnotations.TargetPort);
     }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_DefaultFile_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // defaults to ./dab-config.json which exists in this test project root
+        builder.AddDataAPIBuilder("dab");
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_PortOnly_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        builder.AddDataAPIBuilder("dab", port: 1234);
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_ValidFile_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // file exists in test project root
+        builder.AddDataAPIBuilder("dab", configFilePaths: "./dab-config.json");
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_ValidFileWithPort_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // file exists in test project root
+        builder.AddDataAPIBuilder("dab", port: 1234, configFilePaths: "./dab-config.json");
+    }
```

Comment:
For these would could look at the annotations of the resource and find the `ContainerMountAnnotation` and assert that it contains the right value(s).

Reply:
I am not sure how to do that. Can you point me to the syntax?